akd_core/benches/parallel_vrfs.rs

```diff
@@ -9,13 +9,68 @@
 
 extern crate criterion;
 use self::criterion::*;
+use akd_core::configuration::NamedConfiguration;
 use akd_core::ecvrf::{VRFExpandedPrivateKey, VRFPublicKey};
 use akd_core::VersionFreshness;
 use akd_core::{ecvrf::VRFKeyStorage, AkdLabel, AkdValue};
 use rand::distributions::Alphanumeric;
 use rand::Rng;
 
-fn bench_single_vrf(c: &mut Criterion) {
+macro_rules! bench_config {
+    ( $x:ident ) => {
+        paste::paste! {
+            // NOTE(new_config): Add a new configuration here
+
+            #[cfg(feature = "whatsapp_v1")]
+            fn [<$x _ whatsapp_v1_config>](c: &mut Criterion) {
+                $x::<akd_core::WhatsAppV1Configuration>(c)
```

Comment:
yeah so here we could get rid of the features bit (I think) if you export a common `akd_core::TheConfiguration` or something